File: libmonet/lib/quantize/quantizer.js
'use strict';

export class quantizer_wsmeans_DistanceAndIndex {
    constructor() {
        this.index = this.distance = -1;
    }
}
export class quantizer_wu_Box {
    constructor() {
        this.vol = this.b1 = this.b0 = this.g1 = this.g0 = this.r1 = this.r0 = 0;
    }
}
export class quantizer_wu_CreateBoxesResult {
    constructor(resultCount) {
        this.resultCount = resultCount;
    }
}
export class quantizer_wu_MaximizeResult {
    constructor(cutLocation, maximum) {
        this.cutLocation = cutLocation;
        this.maximum = maximum;
    }
}

File: dart/libmonet/lib/quantize/quantizer.dart
class quantizer_wsmeans_DistanceAndIndex {
  int index = -1;
  num distance = -1;
}

class quantizer_wu_Box {
  int vol = 0, b1 = 0, b0 = 0, g1 = 0, g0 = 0, r1 = 0, r0 = 0;
}

class quantizer_wu_CreateBoxesResult {
  final int resultCount;
  quantizer_wu_CreateBoxesResult(this.resultCount);
}

class quantizer_wu_MaximizeResult {
  final cutLocation;
  final num maximum;
  quantizer_wu_MaximizeResult(this.cutLocation, this.maximum);
}
